docs/arch/relay_op_strategy.rst

```diff
@@ -278,5 +278,5 @@ model to learn which implementation is used for each operator.
 
 .. code:: python
 
-    logging.getLogger("te_compiler").setLevel(logging.INFO)
-    logging.getLogger("te_compiler").addHandler(logging.StreamHandler(sys.stdout))
+    logging.getLogger("tvm.relay.backend.te_compiler").setLevel(logging.INFO)
```

Comment:
i'm not an expert in our doc rendering, but looks like `__name__` doesn't work in this case? if so, this change looks good to me

Reply:
Me too, I think it is example script in which user is configuring logging, so I just updated old logger name to new one.